src/Surfnet/StepupSelfService/SelfServiceBundle/Service/RemoteVetting/SamlCalloutHelper.php

```diff
@@ -120,9 +121,9 @@ public function handleResponse(Request $request, $identityProviderName)
         // Create log DTO in order to store
         $attributeLogDto = new AttributeListDto(
             $assertion->getAttributes(),
-            (string)$subjectConfirmation->NameID,
-            $assertion->toXML()->ownerDocument->saveXML()
+            (string)$subjectConfirmation->NameID
         );
+        // todo: store raw result ($assertion->toXML()->ownerDocument->saveXML())
```

Comment:
Maybe add a link to a ticket where we would be triggered to do this?

Reply:
I don't know if it should be implemented, initially it was implemented to use in the encryptor but it seems not used any more and according to the story it currently seems sufficient.
https://www.pivotaltracker.com/story/show/171571453

Reply:
Thanks for removing the comment, I don't recal we should store the full saml response, just the relevant attributes from the assertion.